jade/distribution.yaml

```diff
@@ -5192,13 +5192,14 @@ repositories:
       - rosserial_msgs
       - rosserial_python
       - rosserial_server
+      - rosserial_test
```

Comment:
Is it intentional to release this package? If it is only used for testing and doesn't need to be released you could blacklist it in the release repo.